config/projects.yml

```
@@ -549,14 +549,63 @@ fuzzing:
       cloud: gcp
       minCapacity: 0
       maxCapacity: 10
+    decision:
+      owner: fuzzing+taskcluster@mozilla.com
+      emailOnError: false
+      imageset: docker-worker
+      cloud: gcp
+      minCapacity: 0
+      maxCapacity: 10
+  hooks:
+    decision:
+      description: Hook to trigger the Fuzzing decision task
+      owner: fuzzing+taskcluster@mozilla.com
+      emailOnError: true
+      schedule:
+        - "0 0 */1 * * *"
+      task:
+        provisionerId: proj-fuzzing
+        workerType: decision
+        payload:
+          image: python:3.7-slim
+          features:
+            taskclusterProxy: true
+          environment:
```

```suggestion
          env:
```

(see [payload docs](https://github.com/taskcluster/docker-worker/blob/296b21cf8668b33fcf0869a2c4355323427d8c3a/schemas/v1/payload.json#L163-L167))